spec/features/store_admin/store_admin_can_change_item_information_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature "As a store manager" do
+  let(:store_admin) { create(:user) }
+  let(:store_admin_role) { create(:store_admin) }
+  let(:store) { create(:store, status: "active") }
+  let(:store_item) { create(:item, store: store) }
+
+  scenario "I can change item information for a store that I manage" do
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(store_admin)
```

Comment:
If this is applied more than once it can be pulled out into a before :each method.